Dockerfile

```diff
@@ -28,24 +27,17 @@ LABEL name="hazelcast/hazelcast-enterprise-openshift-rhel" \
       io.openshift.expose-services="5701:tcp" \
       io.openshift.tags="hazelcast,java8,kubernetes,rhel8"
 
-RUN mkdir -p $HZ_HOME
-RUN mkdir -p $HZ_CP_MOUNT
-WORKDIR $HZ_HOME
-
-ADD hazelcast.xml $HZ_HOME/hazelcast.xml
-ADD start.sh $HZ_HOME/start.sh
-ADD stop.sh $HZ_HOME/stop.sh
-
-# Add licenses
-ADD licenses /licenses
+COPY *.xml *.sh $HZ_HOME/
+COPY licenses $HZ_HOME/licenses
```

Comment:
Licenses must be in the main directory. Otherwise the Red Hat Scan complains with the following message.

![image](https://user-images.githubusercontent.com/2834997/76742978-361ca100-6772-11ea-8d93-8a834e728fc1.png)